spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/iam/IdentityService.java

```diff
@@ -26,27 +27,14 @@ public interface IdentityService {
     /**
      * Obtains a client token encoded as a JWT.
      */
-    Result<TokenRepresentation> obtainClientCredentials(String scope);
+    Result<TokenRepresentation> obtainClientCredentials(String scope, String audience);
```

Comment:
Just a thought, would a single object that represents a claim/properties set be more flexible for the future? For the moment it could contain only `scope` and `audience`, so if an additional field would need to be added (e.g., a configurable duration for expiration) there'll be no need to break the interface again.